technical-reports/format/types.md

```diff
@@ -12,21 +12,42 @@ If an explicit type is set, but the value does not match the expected syntax the
 
 ## Color
 
-Represents a 24bit RGB or 24+8bit RGBA color in the sRGB color space. The `$type` property MUST be set to the string `color`. The `$value` property MUST be a string containing a [hex triplet/quartet](https://www.w3.org/TR/css-color-4/#typedef-hex-color) including the preceding `#` character. To support other color spaces, such as HSL, translation tools SHOULD convert color tokens to the equivalent value as needed.
+Represents a color in the UI. The `$type` property MUST be set to the string `color`. The `$value` property MUST be an object string containing a string `colorSpace`, a numeric `coordinates` array, optional numeric `alpha`, string `colorProfile`, and string `fallback` to support legacy color spaces as needed.
+
+| Key            |      Type      | Required | Description                                                                                                                              |
+| :------------- | :------------: | :------: | :--------------------------------------------------------------------------------------------------------------------------------------- |
+| `colorSpace`   |    `string`    |    Y     | An string representing the color space.                                                                                                  |
+| `colorProfile` |    `string`    |          | String representing url of color profile.                                                                                                |
+| `coordinates`  | `number array` |    Y     | An array of numeric values representing individual color coordinates.                                                                    |
+| `alpha`        |    `number`    |          | An integer or floating-point value representing the numeric value.                                                                       |
+| `fallback`     |    `string`    |          | Unit of distance. Supported values: [HEX 6](https://www.w3.org/TR/css-color-4/#typedef-hex-color) including the preceding `#` character. |
 
 For example, initially the color tokens MAY be defined as such:
 
 <aside class="example">
 
 ```json
 {
-  "Majestic magenta": {
-    "$value": "#ff00ff",
-    "$type": "color"
+  "Hot pink": {
+    "$type": "color",
+    "$value": {
+      "colorSpace": "display-p3",
+      "colorProfile": "http://example.org/display-p3.icc",
```

Comment:
Here and below, the format should match the updated syntax of `colorSpace` + `channels` + `alpha` (optional) + `hex` (optional)

```suggestion
     "channels": [0.92, 0.2, 0.97],
      "hex": "#ff00ff" // HEX 6 supported only
```

Reply:
Thanks for this suggestion! I should clarify why the optional `colorProfile` is listed here. I was thinking of the cases for print where a specific color profile should be associated with the color space for conversion. Perhaps we do a separate callout for this and use CMYK as the color space example.